week-3/Homework/mandatory/7-password-validator.js

```diff
@@ -23,47 +23,49 @@ PasswordValidationResult=  [false, false, false, false, true]
 */
 
 function validatePasswords(passwords) {
-
+  const check = passwords.filter(element => element.includes(element.length >= 5) && element === "A")
 }
 
 /* ======= TESTS - DO NOT MODIFY ===== */
 
 const passwords1 = ["Se%5", "TktE.TJTU", "384HsHF", "dvyyeyY!5", "tryT3729."]
-const passwords2 = ["StUFf27", "PlEnty", "Jai.33" "shajsaUA**&&", "PlEnty"]
+const passwords2 = ["StUFf27", "PlEnty", "Jai.33"
```

Comment:
It looks like you've accidentally removed a comma here which is causing a syntax error.